address_test.go

```diff
@@ -20,18 +20,21 @@ import (
 type CustomParamStruct struct {
 	PubKeyHashAddrID byte
 	ScriptHashAddrID byte
+	Bech32HRPSegwit  string
 }
 
 var CustomParams = CustomParamStruct{
-	PubKeyHashAddrID: 0x30, // starts with L
-	ScriptHashAddrID: 0x32, // starts with M
+	PubKeyHashAddrID: 0x30,  // starts with L
+	ScriptHashAddrID: 0x32,  // starts with M
+	Bech32HRPSegwit:  "ltc", // starts with ltc
 }
 
 // We use this function to be able to test functionality in DecodeAddress for
 // defaultNet addresses
 func applyCustomParams(params chaincfg.Params, customParams CustomParamStruct) chaincfg.Params {
 	params.PubKeyHashAddrID = customParams.PubKeyHashAddrID
 	params.ScriptHashAddrID = customParams.ScriptHashAddrID
+	params.Bech32HRPSegwit = customParams.Bech32HRPSegwit
```

Comment:
This is just missing a call to `chaincfg.Register`. You need to do something like this:

```go
if err := chaincfg.Register(&params); err != nil {
        panic(err)
}
```

You may also need to add the following to ensure that there are no duplicate network parameters, since we copy the config from `chaincfg.MainNetParams`.

```golang
params.Net = 0xdeadbeef  // whatever the value is for Litecoin
```